### PR TITLE
Typo fix: Triple dot to ellipsis

### PR DESCRIPTION
**Pull request by rakekniven**
Signed-off-by: Mark Ziegler <mark.ziegler@rakekniven.de>

### CODE REVIEW

File: apps/dav/lib/CalDAV/Schedule/IMipPlugin.php [hunk 1]
```diff
@@ -501,7 +501,7 @@ private function addResponseButtons(IEMailTemplate $template, IL10N $l10n,
 			'token' => $token,
 		]);
 		$html = vsprintf('<small><a href="%s">%s</a></small>', [
-			$moreOptionsURL, $l10n->t('More options ...')
+			$moreOptionsURL, $l10n->t('More options…')
```
Comment (nickvergessen) [inline apps/dav/lib/CalDAV/Schedule/IMipPlugin.php]: there should be a space between options and … because it's not the word options that is shortened

Comment (rakekniven) [reply apps/dav/lib/CalDAV/Schedule/IMipPlugin.php]: The holy grail of grammar. AFAIK there is no general rule for this. We hat various discussions on Transifex and here in the past.
@comradekingu Space or no space? Can you enlighten us?

Comment (nickvergessen) [reply apps/dav/lib/CalDAV/Schedule/IMipPlugin.php]: The golden rule was set up by our designer @jancborchardt :
* No space if you shorten a word
* Space when you shorten a sentence

Comment (rakekniven) [reply apps/dav/lib/CalDAV/Schedule/IMipPlugin.php]: Where to put this golden rule?
Every dev needs to know this.
Every translator needs this to know as well.

Comment (nickvergessen) [reply apps/dav/lib/CalDAV/Schedule/IMipPlugin.php]: https://github.com/nextcloud/server/wiki/Translation-knowledge-(valid-for-whole-nextcloud-project)#ellipsis

**Review by jancborchardt — APPROVED** [review]
Good detail fix, sure :)

**Review by jospoortvliet — APPROVED** [review]
Looks good now :rocket:

**Comment by jospoortvliet** [comment]
if I read it correctly, drone complains it isn't signed off, but it is, right?

**Comment by MariusBluem** [comment]
> if I read it correctly, drone complains it isn't signed off, but it is, right?

unfortunately not ... the second commit is not signed off :(

**Comment by jospoortvliet** [comment]
Sjees, it is that difficult? Ok... pfff ;-) Should I just override and merge?

**Pull request merged**